spec/03-plugins/17-jwt/fixtures.lua

```diff
@@ -39,6 +39,46 @@ FLiGOm5uTMEk8S4txs2efueg1XyymilCKzzuXlJvrvPA4u6HI7qNvuvkvUjQmwBH
 gwIDAQAB
 -----END PUBLIC KEY-----
 ]],
+    rs512_private_key = [[
```

Comment:
The spaces at the beginning of this line are undesired; other keys are not indented like so.